addon/mixins/touch-action.js

```diff
@@ -37,15 +36,15 @@ export default Mixin.create({
   ignoreTouchAction: false,
 
   init() {
-    this._super();
+    this._super(...arguments);
 
     const {
       tagName,
       ignoreTouchAction,
       click
     } = this;
 
-    const hasClick = click !== K;
+    const hasClick = typeof click === 'function';
```

Comment:
I forget what exactly but there was an issue with `if (this.click)`, note, we aren't actually executing the function, we're just inspecting the reference to it, so what it returns doesn't matter.  `K` returns `this` just in case you do invoke it as a nicety for chaining.